### ISSUE TITLE
The bot didnt catch with powerballs/ultraballs.

### ISSUE THREAD

**Issue by MettheRZ** [PokemonGoF/PokemonGo-Bot#395]
The bot didnt catch with powerballs/ultraballs. It does not detect better balls than pokeballs.

**Comment by Notnuk** [comment]
Same problem.

**Comment by vincesmits** [comment]
u should explain your problem a bit better.

How many of each pokeball did you have and what was the cp value of the pokemons that you where expecting to be caught with great/ultra balls?

**Comment by MettheRZ** [comment]
[#] PokeBalls: 0
[#] GreatBalls: 51
[#] UltraBalls: 18

**Comment by MettheRZ** [comment]
and bot didnt catch with greatballs and ultraballs

**Comment by Witzbold** [comment]
I have the same problem. The bot seems to primarily use pokeballs. I realized that sometimes also superballs are used. But although I had many superballs the I got the following log:

[#] A Wild Paras appeared! [CP 60] [Potential 0.42]
[x] Out of pokeballs, switching to farming mode...
12
[#] A Wild Pidgey appeared! [CP 40] [Potential 0.27]
[x] Out of pokeballs, switching to farming mode...

But I have also seen the bot using superballs although I can't find the log for that.
